packages/manager/modules/pci/src/projects/project/storages/cold-archive/add/add.routing.js

```diff
@@ -11,6 +13,21 @@ export default /* @ngInject */ ($stateProvider) => {
         $translate.instant(
           'pci_projects_project_storages_cold_archive_add_breadcrumb',
         ),
+      trackAddContainerPage: /* @ngInject */ (atInternet) => (action) => {
+        const hit = `${COLD_ARCHIVE_TRACKING.ADD.PREFIX}::${action}`;
+        atInternet.trackPage({
+          name: hit,
+          type: 'page',
+        });
+      },
+
+      trackAddContainerClick: /* @ngInject */ (atInternet) => (action) => {
+        const hit = `${COLD_ARCHIVE_TRACKING.ADD.PREFIX}::${action}`;
+        atInternet.trackClick({
+          name: hit,
+          type: 'click',
+        });
+      },
```

Comment:
**nitpick:** This function can be defined in the controller file.

Reply:
Done.